### DISCUSSION TITLE
Job Requirements

### DISCUSSION THREAD

**Discussion by Appl3Tree** [ericgrandt/TotalEconomy#394] [Ideas]
Hello, I'm trying to set up some custom jobs that have higher pay per action & salary but would require combinations of other jobs to be at certain levels. Is it possible to require multiple jobs at different levels though? I'd like to have some sort of tiering of the jobs so my users can work their way up to making money easier.

I know there is the option in the jobs.conf to have the following code:
```
require {
    job=example1
    lvl=5
    permission="example.permission"
}

**Comment by ericgrandt** [comment]
This is not currently possible, but this could be something that's added in the API 8 version of Total Economy. I'm going to categorize this thread as an "Idea" so we can keep it in mind.
